contents/blog/missing-recordings.md

```diff
@@ -0,0 +1,56 @@
+---
+date: 2021-06-10
+title: The mystery of missing recordings at PostHog
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/
+featuredImageType: full
+author: ["rick-marron"]
+categories: ["Engineering"]
+---
+
+Session Recording is one of PostHog's core features. It allows customers to see their website being used, mouse clicks and all. It's really cool, but, until recently, we had one big issue with the feature. Customers would often tell us recordings were missing, and we didn't know why.
```

Comment:
I think the intro should talk a little more about the Session Recording feature. How does it work, how do people use it etc. We want to talk up the feature as much as possible.